roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4531460&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4531460&tab=artifacts#/activerecord) on master @ [e7e596ede9d3da877e313a3d39ab498a5cb7d27b](https://github.com/cockroachdb/cockroach/commits/e7e596ede9d3da877e313a3d39ab498a5cb7d27b):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-210-ge7e596ede9
		Tests run against activerecord 6.1
		6640 Total Tests Run
		6639 tests passed
		1 test failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77322 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-13631

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -28,7 +28,7 @@ import (
 var activerecordResultRegex = regexp.MustCompile(`^(?P<test>[^\s]+#[^\s]+) = (?P<timing>\d+\.\d+ s) = (?P<result>.)$`)
 var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)\.?(?P<subpoint>\d*)$`)
 var supportedRailsVersion = "6.1"
-var activerecordAdapterVersion = "v6.1.6"
+var activerecordAdapterVersion = "v6.1.8"
 
 // This test runs activerecord's full test suite against a single cockroach node.
 

--- a/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
+++ b/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
@@ -411,7 +411,6 @@ var pgjdbcBlockList22_1 = blocklist{
 	"org.postgresql.test.jdbc2.ResultSetTest.testRowResultPositioning":                                                                                                         "unknown",
 	"org.postgresql.test.jdbc2.ResultSetTest.testgetBadBoolean":                                                                                                                "32552",
 	"org.postgresql.test.jdbc2.SearchPathLookupTest.testSearchPathNormalLookup":                                                                                                "unknown",
-	"org.postgresql.test.jdbc2.ServerCursorTest.testBasicFetch":                                                                                                                "41412",
 	"org.postgresql.test.jdbc2.ServerCursorTest.testBinaryFetch":                                                                                                               "41412",
 	"org.postgresql.test.jdbc2.ServerErrorTest.testCheckConstraint":                                                                                                            "27796",
 	"org.postgresql.test.jdbc2.ServerErrorTest.testColumn":                                                                                                                     "27796",

